packages/dhis-api/src/DhisApi.js

```diff
@@ -35,6 +35,7 @@ const LATEST_LOOKBACK_PERIOD = '600d';
 
 export class DhisApi {
```

Comment:
The changes in this class are just to allow the api to take ids instead of codes.